k8s/live/azure-dev/postgresql-configmap.yaml

```
@@ -0,0 +1,36 @@
+      create database "dev_cgmes_boundary";
+      create database "dev_iidm";
+      create database "dev_import_history";
+      create database "dev_cgmes_assembling";
+      create database "dev_useradmin";
+      create database "dev_sensitivityanalysis";
+      create database "dev_shortcircuit";
+      create database "dev_case_metadata";
+      create database "dev_timeseries";
+      create database "dev_voltageinit";
+      create database "dev_loadflow";
+    EOF
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)